### PR TITLE
[Clojure] Add numbers exercise

### PR DESCRIPTION
**Pull request by porkostomus**

### CODE REVIEW

File: languages/clojure/exercises/concept/numbers/.meta/example.clj [hunk 1]
```diff
@@ -0,0 +1,21 @@
+(ns numbers)
+
+(def default-rate 221)
+
+(defn success-rate [speed]
+  (cond (= speed 10) 0.77
+        (= speed 9) 0.8
```
Comment (bemself) [inline languages/clojure/exercises/concept/numbers/.meta/example.clj:7]: @porkostomus hah I was thinking what happens when speed is between 9 and 10...and found out the table at the beginning didn't mention that, so the speed here is supposed to be integers only, that should be fine

**Comment by ErikSchierboom** [comment]
@porkostomus Is there anything I can do to help with this PR?

File: languages/clojure/exercises/concept/numbers/.meta/example.clj [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "Returns the assembly line's production rate per hour,
+   taking into account its success rate"
+  [speed]
+  (* (* default-rate speed)
+     (success-rate speed)))
+
+(defn working-items
+  "Calculates how many working cars are produced per minute"
+  [speed]
+  (int (quot (production-rate speed) 60)))
```
Comment (ErikSchierboom) [inline languages/clojure/exercises/concept/numbers/.meta/example.clj:21]: I think the introduction should mention that casting to an `int` is done by using a function with the same name. I think the F# exercise does not explain this (this is a mistake).

**Review by ErikSchierboom — APPROVED** [review]
Awesome!

**Pull request merged**